cfgov/cfgov/settings/production.py

```diff
@@ -1 +1,27 @@
 from .base import *
+
+# Sends an email to developers in the ADMIN_EMAILS list if Debug=False for errors
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse'
```

Comment:
not exactly sure, but its part of the syntax they use to instantiated filters https://docs.djangoproject.com/en/1.9/topics/logging/#django.utils.log.RequireDebugFalse